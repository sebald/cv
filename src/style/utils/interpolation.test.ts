import { propOrEmpty } from './interpolation';

test('prop or empty', () => {
  const fooOrEmpty = propOrEmpty('foo');

  expect(fooOrEmpty({} as any)).toMatchSnapshot();
  expect(fooOrEmpty({ foo: 'bar' } as any)).toMatchSnapshot();
  expect(fooOrEmpty({ notFoo: ':(' } as any)).toMatchSnapshot();
});
