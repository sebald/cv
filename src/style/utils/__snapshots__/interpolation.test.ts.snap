// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prop or empty 1`] = `Object {}`;

exports[`prop or empty 2`] = `
Object {
  "foo": "bar",
}
`;

exports[`prop or empty 3`] = `Object {}`;
